spark/v3.1/spark/src/main/java/com/netease/arctic/spark/SparkSQLProperties.java

```diff
@@ -31,4 +31,8 @@ public class SparkSQLProperties {
   public static final String REFRESH_CATALOG_BEFORE_USAGE = "spark.sql.arctic.refresh-catalog-before-usage";
 
   public static final String REFRESH_CATALOG_BEFORE_USAGE_DEFAULT = "false";
+
+  public static final String CHECK_DATA_DUPLICATES_ENABLE = "spark.sql.check-data-duplicates.enabled";
+
+  public static final String CHECK_DATA_DUPLICATES_ENABLE_DEFAULT = "true";
```

Comment:
Should we change default value to `false`,  considering the big cost of this operation?

Reply:
If we want to make the primary key unique by default, does it means this parameter should also be enabled by default?